test/app/digest.skipcond

```diff
@@ -3,5 +3,11 @@ import platform
 # Disabled on OpenBSD due to fail #XXXX.
 if platform.system() == 'OpenBSD':
     self.skip = 1
+# Disabled for macOS/M1 due to the #6097 issue.
+elif platform.system() == 'Darwin' and platform.machine() == 'arm64'
+    self.skip = 1
```

Comment:
We can just disable JIT (`jit.off()`) in the test. No reason to skip testing of the entire module.

Reply:
Ok, fixed.